### PR TITLE
Question 6

### PR DESCRIPTION
**Pull request by taruns52**
Let me know if I can improve any better. Thank you in Advance for reviewing

### CODE REVIEW

File: Question_6.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+sub_str_len = len(substring)
+str_len = len(string)
+
+count = 0
+
+for i in range(str_len - sub_str_len + 1):
+    if string[i:i+sub_str_len] == substring:
+        count+=1
+
+print('Total occurance of sub-string',substring,'in',string,'is',count)
```
Comment (himanshu-suthar-tagline) [inline Question_6.py:19]: Which solution is better and why?

Comment (himanshu-suthar-tagline) [reply Question_6.py:19]: We don't want to keep two solutions, we just need one solution.

Comment (taruns52) [reply Question_6.py:19]: > Which solution is better and why?

the second one is better 

> We don't want to keep two solutions, we just need one solution.

okay sir

Comment (himanshu-suthar-tagline) [reply Question_6.py:19]: Please remove one of the solutions.

**Comment by himanshu-suthar-tagline** [comment]
Please merge this.

**Pull request merged**